### PR TITLE
Auto-detect carrier’s country code

### PR DESCRIPTION
**Pull request by Erin-Mounts**
also update mPower 2 code to work with resolved BridgeApp/SageResearch lib versions

### CODE REVIEW

**Comment by Erin-Mounts** [comment]
@dephillipsmichael it won't let me tag you as a reviewer but I'd like you to have a look if you can, thanks

File: mPower2/mPower2/SignInTaskViewController.swift [hunk 1]
```diff
@@ -54,6 +55,17 @@ class SignInTaskViewController: RSDTaskViewController, SignInDelegate {
     var regionCode: String? = "US" // TODO: emm 2018-04-25 Handle non-US phone numbers for international studies
 
     init() {
+        if #available(iOS 12.0, *) {
```
Comment (dephillipsmichael) [inline mPower2/mPower2/SignInTaskViewController.swift:58]: This file change looks good.  Curious, how do you test this for Netherlands?

**Comment by dephillipsmichael** [comment]
Do you mind removing all the other files changes besides the one to SignInTaskViewController? I feel like it's going to cause a bunch of conflicts with my branch.

**Review by dephillipsmichael — APPROVED** [review]
Approved, but would prefer to have only sign in changes pushed

**Review by syoung-smallwisdom — APPROVED** [review]
looks good

File: mPower2/mPower2/SignInTaskViewController.swift (hunk 1 above)
Comment (Erin-Mounts) [reply mPower2/mPower2/SignInTaskViewController.swift:58]: I have not found any way to do so, short of trying it on a NL phone or getting a SIM card from a NL carrier and swapping it in to your phone. I tested it for US and am basically going on faith that the iOS API works as described in other countries.

**Comment by Erin-Mounts** [comment]
> Do you mind removing all the other files changes besides the one to SignInTaskViewController? I feel like it's going to cause a bunch of conflicts with my branch.

@dephillipsmichael  I'd be OK with that in principle, but Larsson wants this in an iOS build for Nigeria ASAP (which is based on the US version rather than the NL one). Can you test what happens if you try to merge with your branch or rebase your branch on top of this one? (or point me at your branch and I'll test it and see if I can do anything to make it go smoothly.)

**Pull request merged**